crates/bevy_asset/src/asset_server.rs

```diff
@@ -45,6 +45,7 @@ fn format_missing_asset_ext(exts: &[String]) -> String {
 pub(crate) struct AssetRefCounter {
     pub(crate) channel: Arc<RefChangeChannel>,
     pub(crate) ref_counts: Arc<RwLock<HashMap<HandleId, usize>>>,
+    pub(crate) mark_unused_assets: Arc<RwLock<Vec<HandleId>>>,
```

Comment:
Both times `mark_unused_assets` is used, it's with exclusive (write) access.
If read access is not explicit requires, it's probably best to use `Mutex` instead of `RwLock`.